indexers/keyvalue/RedisDBIndexer/tests/test_redisdbindexer.py

```diff
@@ -0,0 +1,49 @@
+import os
+import shutil
+
+import jina.proto.jina_pb2 as jina_pb2
+from google.protobuf.json_format import MessageToJson
+from jina.executors.indexers import BaseIndexer
+
+from .. import RedisDBIndexer
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+def rm_files(file_paths):
```

Comment:
we are using `metas` now, see draft for inspiration https://github.com/jina-ai/jina-hub/pull/424